thirdeye-ui/package.json

```diff
@@ -134,6 +134,7 @@
         "html-webpack-plugin": "5.3.2",
         "husky": "7.0.2",
         "identity-obj-proxy": "^3.0.0",
+        "iso8601-duration": "^2.1.1",
```

Comment:
We should be using absolute versions. Sometimes, breaking changes end up being introduced in minor versions and things break.